portal/i18next_gulpfile.js

```diff
@@ -0,0 +1,163 @@
+/*
+ * prerequisites:
+ * nodeJS virtual environment install via pip nodeenv
```

Comment:
Cool find (nodeenv)! Could you add some notes about installing/running the extraction process to our [existing i18n notes](https://github.com/uwcirg/true_nth_usa_portal/blob/master/docs/source/development.rst#internationalization-i18n)?

Reply:
@ivan-c Ivan, I will.  In talking to Adam,  I think the translation part of the documentation will need to be updated to reflect the smartling uploading process.   I will wait until you get back so we can update the documentation properly.